src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java

```diff
@@ -78,9 +80,9 @@ public void update(Collection<Sample> samples, AnalysisSubmission analysis) thro
 		Path filePath = aof.getFile();
 
 		Map<String, MetadataEntry> stringEntries = new HashMap<>();
-		try {
-			@SuppressWarnings("resource") String jsonText = new Scanner(
-					new BufferedReader(new FileReader(filePath.toFile()))).useDelimiter("\\Z")
+
+		try(InputStream inputStream = iridaFileStorageUtility.getFileInputStream(filePath)) {
+			@SuppressWarnings("resource") String jsonText = new Scanner(inputStream).useDelimiter("\\Z")
```

Comment:
Actually thinking about this a bit more, I think this can be done `IridaFiles` instead of the `iridaFileStorageUtility`?  That way we don't need to wire a new dependency into every updater class.

Reply:
Updated the built in sample updaters to use IridaFiles in eb2d3b4fd2ef1fc91ed99b94fb0574c6bc579ea9